apps/judicial-system/web/src/routes/Court/CourtRecord/CourtRecord.spec.tsx

```diff
@@ -29,12 +29,15 @@ describe('/domari-krafa/thingbok', () => {
           ...mockJudgeQuery,
           ...mockUpdateCaseMutation(
             [
+              {
```

Comment:
Why is this needed?

Reply:
Because prosecutorDemands is autofilled from demands.

The logic goes like this:
1.  The prosecutor fills in requestedValidToDate
2. In the prosecutor flow, the client autfills demands using formatPresecutorDemands
3. In the court flow, the client autofills prosecutorDemands using demands

Reply:
I guess we no longer need the second prosecutorDemands entry. Will fix.